src/main/resources/site/parts/article/article.html

```diff
@@ -36,9 +36,10 @@ <h1 class="col-lg-10" data-th-text="${title}" />
             <div class="article-body" data-th-utext="${bodyText}" />
         </div>
         <div class="col-12 p-0">
+            <div data-th-if="${#lists.size(externalLinkConfig)}" class="divider-wrapper" id="dividerId" /> <!-- TODO: Change if-statement when the 'Related Statistics' portion is implemented-->
```

Comment:
Why hide this if there's no externalLinks?

Reply:
At the time, the only related content was the external links so the solution is outdated per today. The external links portion would be hidden when there are no inputs in the article configuration, while the divider remained.  Ideally, I wanted divider to be hidden when there's nothing beneath it, but I couldn't quite think of a better solution. 🤔

Reply:
I've thought to use the :last-child class but unfortunately the divider would still be inserted, just invisible I've also considered the index.last thymeleaf solution but there were no lists.

Reply:
css is probably the easiest solution. I think we're going to add dividers between each of the related sections as well, and because it's tricky to keep track of what's happening in other parts